src/Core/Geographic/Coordinates.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: Coordinates
+ * Description: Coordonnées cartographiques
+ */
+
+import * as THREE from 'three';
+import mE from '../Math/MathExtended';
+import Ellipsoid from '../Math/Ellipsoid';
+
+export function ellipsoidSizes() {
+    return {
+        x: 6378137,
+        y: 6356752.3142451793,
+        z: 6378137,
+    };
+}
+
+const ellipsoid = new Ellipsoid(ellipsoidSizes());
+
+export const UNIT = {
+    RADIAN: 0,
+    DEGREE: 1,
+    METER: 2,
+};
+
+export const supportedCRS = {
+    'EPSG:4326': UNIT.DEGREE,
+    'EPSG:4326R': UNIT.RADIAN,
```

Comment:
Yes, hence my comment https://github.com/iTowns/itowns2/pull/253#issuecomment-282790796.
Any better suggestion ?
The  alternative I had in mind is to simply not support EPSG:4326 + radians. In this case the user will have to convert to degrees on its own.